modules/distribution/resources/migration-2.6.0_to_3.0.0/db2.sql

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE AM_SYSTEM_APPS (
+       ID INTEGER NOT NULL,
+       NAME VARCHAR(50) NOT NULL,
+       CONSUMER_KEY VARCHAR(512) NOT NULL,
+       CONSUMER_SECRET VARCHAR(512) NOT NULL,
+       CREATED_TIME TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+       PRIMARY KEY (ID)
+)
+/
+CREATE SEQUENCE AM_SYSTEM_APPS_SEQUENCE START WITH 1 INCREMENT BY 1 NOCACHE
+/
+CREATE TRIGGER AM_SYSTEM_APPS_TRIGGER NO CASCADE BEFORE INSERT ON AM_SYSTEM_APPS
+REFERENCING NEW AS NEW FOR EACH ROW MODE DB2SQL
+  BEGIN ATOMIC
+    SET (NEW.ID)
+    = (NEXTVAL FOR AM_SYSTEM_APPS_SEQUENCE);
+  END
+/
+
+CREATE TABLE AM_API_CLIENT_CERTIFICATE (
```

Comment:
Shouldn't we use create table if not exist?

Reply:
DB2 does not support IF NOT EXISTS statement. This check is not there on other DB2 scrips as well..